crates/bevy_input/src/gamepad.rs

```diff
@@ -307,12 +307,13 @@ pub enum ButtonSettingsError {
     },
 }
 
-/// Stores a connected gamepad's state and any metadata such as the device name.
+/// The [`Gamepad`] [`component`](Component) stores a connected gamepad's metadata such as the `name` and its [`GamepadButton`] and [`GamepadAxis`].
 ///
-/// An entity with this component is spawned automatically after [`GamepadConnectionEvent`]
-/// and updated by [`gamepad_event_processing_system`].
+/// The [`entity`](Entity) representing a gamepad and its [`minimal components`](GamepadSettings) are automatically managed.
 ///
-/// See also [`GamepadSettings`] for configuration.
+/// # Usage
+///
+/// The only way to obtain a [`Gamepad`] is by [`query`](Query).
```

Comment:
It's obvious that components obtained via query, so maybe keep the old doc?

```suggestion
/// See also [`GamepadSettings`] for configuration.
```